### PR TITLE
Insight price and daily tx charts w/ Nav changes

### PR DESCRIPTION
**Pull request by bsnowden3**

### CODE REVIEW

File: packages/insight/src/components/footer/footer.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <div class="pages">
       <a *ngIf="chainNetwork" (click)="openPage('broadcast-tx')">Broadcast Transaction</a>
        &middot; 
+      <a *ngIf="chainNetwork" (click)="openPage('messages')">Verify Message</a>
```
Comment (gabrielbazan7) [inline packages/insight/src/components/footer/footer.html]: You can remove this line

**Comment by bsnowden3** [comment]
Needs #2378

**Comment by micahriggan** [comment]
Let's merge this after BWS with ETH rates is merged

**Comment by bsnowden3** [comment]
Needs #2402

File: packages/insight/src/components/daily-transaction-chart/daily-transaction-chart.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+daily-transaction-chart{
```
Comment (micahriggan) [inline packages/insight/src/components/daily-transaction-chart/daily-transaction-chart.scss]: probably don't need this file

File: packages/insight/src/providers/blocks/blocks.ts [hunk 3]
```diff
@@ -142,13 +154,15 @@ export class BlocksProvider {
...
+    const url = `${this.api.getUrl(
+      chainNetwork
+    )}/block?since=${since}&limit=${numBlocks}&paging=height&direction=-1`;
     return this.httpClient.get<ApiEthBlock[] & ApiUtxoCoinBlock[]>(url);
   }
 
   public getBlock(
     hash: string,
-    chainNetwork: ChainNetwork
+    chainNetwork: ChainNetworkgi
```
Comment (micahriggan) [inline packages/insight/src/providers/blocks/blocks.ts]: Typo

**Comment by micahriggan** [comment]
npm i --save chart.js

**Comment by micahriggan** [comment]
Looks good, other than comments

**Pull request merged**